From a4007f7d0bdfb98457487c85304188c411e6d991 Mon Sep 17 00:00:00 2001
From: Sibo Dong <46512211+dongsibo@users.noreply.github.com>
Date: Thu, 23 Mar 2023 13:08:12 +0900
Subject: [PATCH] ui/gtk3: Update Emoji shortcut key in ibus-emoji(7)

The default Emoji shortcut key was changed but not updated in the
ibus-emoji.7 man page.

Fixes: https://github.com/ibus/ibus/commit/1520c39

BUG=https://github.com/ibus/ibus/pull/2475
---
 ui/gtk3/ibus-emoji.7.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gtk3/ibus-emoji.7.in b/ui/gtk3/ibus-emoji.7.in
index b4d941eca..31074300b 100644
--- a/ui/gtk3/ibus-emoji.7.in
+++ b/ui/gtk3/ibus-emoji.7.in
@@ -51,7 +51,7 @@ E.g. "Noto Color Emoji", "Android Emoji" font.
 
 .SH "KEYBOARD OPERATIONS"
 .TP
-\fBControl-Period or Control-Semicolon\fR
+\fBSuper-Period\fR
 Launch IBus Emojier. The shortcut key can be customized by
 .B ibus\-setup (1).
 .TP
